packages/esm-patient-queues-app/src/active-visits/active-visits-tab.component.tsx

```diff
@@ -32,48 +32,9 @@ function ActiveVisitsTabs() {
   const session = useSession();
 
   const getTabStatus = (selectedIndex) => {
-    return selectedIndex === 0 ? '' : 'COMPLETED';
+    return selectedIndex === 0 ? '' : 'completed';
```

Comment:
Create an object to hold the statuses.
`export const QueueStatus = {
     Completed: "completed",
     Pending: "pending"
    }; 
`